tests/k8s/1-034_validate_webhook_notifications/00-setup-webhook-server.yaml

```diff
@@ -0,0 +1,168 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook
+spec:
+  selector:
+    app: webhook
+  ports:
+    - name: https
+      port: 443
+      targetPort: 9000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook
+  labels:
+    app: webhook
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: webhook
+  template:
+    metadata:
+      labels:
+        app: webhook
+    spec:
+      containers:
+        - name: server
+          image: quay.io/svghadi/webhook-server:latest
```

Comment:
Should we move this image to `quay.io/argoprojlabs/` similar to `quay.io/argoprojlabs/argocd-notifications-e2e-smtplistener:multiarch` which is used in `1-022_validate_notifications`?